src/extension.ts

```diff
@@ -0,0 +1,83 @@
+import path from "path";
+import { fs } from "vortex-api";
+import { GOGAPP_ID, STEAMAPP_ID, EPICAPP_ID } from './index.metadata';
+import { promptUserInstallREDmodDLC } from "./ui.dialogs";
+import {
+  VortexApi,
+  VortexDiscoveryResult,
+  VortexExtensionContext,
+  vortexUtil,
+} from "./vortex-wrapper";
+
+// This function runs on starting up Vortex or switching to Cyberpunk as the active game. This may need to be converted to a test, but the UI for tests is less flexible.
+
+const getREDmodetails = (id: string): { name: string, url: string } => {
+  if (!id || ![`epic`, `gog`, `steam`].includes(id)) return { name: undefined, url: `https://www.cyberpunk.net/en/modding-support` };
+
+  const gameStoreData = {
+    epic: {
+      name: `Epic Games Store`,
+      url: `https://store.epicgames.com/en-US/p/cyberpunk-2077`,
+    },
+    steam: {
+      name: `Steam`,
+      url: `steam://run/2060310`,
+    //   url: `https://store.steampowered.com/app/2060310/Cyberpunk_2077_REDmod/`,
```

Comment:
Is this needed for some context? If not, nix it
```suggestion
```

Reply:
I think it is nice to see the context.... but steam should be fine to handle it, ya?